README.md

```diff
@@ -28,10 +28,14 @@ cmake --version
 gcc --version
 ```
 
-Add the dependencies necessary to build *TraceLogging* as described at [github.com/microsoft/tracelogging/](https://github.com/microsoft/tracelogging/blob/master/README.md#dependencies).
-
-Install and configure the *Azure IoT Identity Service (AIS)* package as described at [azure.github.io/iot-identity-service/](https://azure.github.io/iot-identity-service/).
-
+Building OSConfig requires the following dependencies (included as submodules). They can be installed manually or included as part of the checkout as a submodule
+#### Manually:
+ * Add the dependencies necessary to build *TraceLogging* as described at [github.com/microsoft/tracelogging/](https://github.com/microsoft/tracelogging/blob/master/README.md#dependencies).
+
+ * Install and configure the *Azure IoT Identity Service (AIS)* package as described at [azure.github.io/iot-identity-service/](https://azure.github.io/iot-identity-service/).
+#### Submodule:
+ * Make sure to clone the repository with submodules - `git clone --recurse-submodules ...` , or
+ * Add submodules to an existing checkout - `git submodule update --init --recursive`

```

Comment:
Please revert back to original. We are looking to make git submodules update persistent so users would not have to do it manually.